roles/nuage_master/tasks/main.yaml

```diff
@@ -33,6 +33,14 @@
 
 - include: certificates.yml
 
+- name: Install Nuage VSD user certificate
+  become: yes
+  copy: src="{{ vsd_user_cert_file }}" dest="{{ cert_output_dir }}/{{ vsd_user_cert_file | basename }}"
```

Comment:
Where is `vsd_user_cert_file` defined?  It could be helpful to document where/how, possibly in the role README.